waggle-dance-integration-tests/src/test/java/com/hotels/bdp/waggledance/WaggleDanceIntegrationTest.java

```diff
@@ -172,9 +169,8 @@ public void run() {
   private Federations stopServerAndGetConfiguration() throws Exception, FileNotFoundException {
     runner.stop();
     // Stopping the server triggers the saving of the config file.
-    Federations federations = YamlFactory
-        .newYaml()
-        .loadAs(new FileInputStream(runner.federationConfig()), Federations.class);
+    Federations federations = YamlFactory.newYaml().loadAs(new FileInputStream(runner.federationConfig()),
```

Comment:
Same here. I just checked this class in my IDE (Eclipse Photon) and the formatting doesn't change like this.

Reply:
Try Photon, I think most of us are using it now. It shouldn't make a difference but you never know. Either way at least you'll be running the latest version of Eclipse and we'll know ;)

Reply:
I'm on photon and I also see those changes like @abhimanyugupta07 , there must be some setting screwing it up